packages/api-page-builder/src/graphql/crud/system.crud.ts

```diff
@@ -204,6 +205,13 @@ export const createSystemCrud = (params: CreateSystemCrudParams): SystemCrud =>
                         notFound: notFoundPage.pid
                     }
                 });
+
+                // Create folder link for initialPages
+                await Promise.all(
+                    initialPagesData.map(page =>
+                        context.folders.createLink({ id: page.id, folderId: "ROOT" })
+                    )
+                );
```

Comment:
I would move this to a plugin, like `context.pageBuilder.onSystemAfterInstall` and do this there, just to keep the logic isolated into a separate file which is easy to unplug if necessary.